components/phenotype-suggest-service/api/src/main/java/org/phenotips/solr/PhenotypeSuggestService.java

```diff
@@ -19,32 +19,35 @@
 
```

Comment:
The package is wrong, this isn't about `solr`, but about phenotypes. @marta- what's a good name for the package? `org.phenotips.diagnosis.differentialPhenotypes` perhaps?